frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1535,3 +1544,14 @@ func summarizeHeredoc(doc string) string {
 	}
 	return summary
 }
+
+func commonImageNames() []string {
+	repos := []string{
+		"alpine", "busybox", "centos", "debian", "golang", "ubuntu", "fedora",
```

Comment:
CentOS is about to die soon :)

Reply:
(I noticed this is used only for typo-checking, so it's fine to retain `centos` here)